### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by 7iomka** [rushabhgediya38/django_dropzone#1]
Hi!
Thank you for demo code - it's very helpful!
My question: It's possible to create a post with multiple images attached?
How I know, each image is `ImageField`
At the time of creating a post, I would like that after each copy of the image has been created - so that all attached images are displayed inside the post (now only 1 image is displayed there)

**Comment by rushabhgediya38** [comment]
Yes it's possible. Here is my GitHub like to create what you want with
description.
https://github.com/rushabhgediya38/django_dropzone

**Comment by rushabhgediya38** [comment]
Try for loop to extract images from dropzone.
for ex: 

class Post(models.Model):
    zip_codes = models.IntegerField(null=True)
    address_user = models.CharField(max_length=400, null=True)
    title = models.CharField(max_length=200)
    phone = models.CharField(max_length=14, null=True, blank=True)
    author = models.ForeignKey(User, on_delete=models.CASCADE)

class Images(models.Model):
    post = models.ForeignKey(Post, on_delete=models.CASCADE, related_name='post_images')
    image = models.ImageField(upload_to=upload_path, null=True, blank=True)

here, image(ImageField) to upload multiple images using dropzone.
You can extract there images using for loop like this.

{% for i in post.post_images.all %}
  {{ i.image.url }}
{% endfor %}

so, you get all images of that particular post.
